src/main/java/com/epam/reportportal/jobs/service/project/ProjectService.java

```diff
@@ -0,0 +1,25 @@
+package com.epam.reportportal.jobs.service.project;
+
+import com.epam.reportportal.jobs.repository.ProjectRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+
+/**
+ * @author <a href="mailto:ivan_budayeu@epam.com">Ivan Budayeu</a>
+ */
+@Service
```

Comment:
no need to use 3 level arch here. classes are pretty simple, so we can use everything in job class